src/catalog/database_catalog.cpp

```diff
@@ -1883,6 +1897,22 @@ void DatabaseCatalog::BootstrapProcContexts(const common::ManagedPointer<transac
                                                            execution::ast::Builtin::NpRunnersDummyReal, true);
   SetProcCtxPtr(txn, postgres::NP_RUNNERS_DUMMY_REAL_PRO_OID, func_context);
   txn->RegisterAbortAction([=]() { delete func_context; });
+
+  func_context = new execution::functions::FunctionContext(
+      "extractYear", type::TypeId::INTEGER, {type::TypeId::INTEGER}, execution::ast::Builtin::ExtractYear);
+  txn->RegisterAbortAction([=]() { delete func_context; });
+  SetProcCtxPtr(txn, postgres::EXTRACT_YEAR_PRO_OID, func_context);
+
+  func_context = new execution::functions::FunctionContext(
+      "like", type::TypeId::BOOLEAN, {type::TypeId::INVALID, type::TypeId::VARCHAR, type::TypeId::VARCHAR},
```

Comment:
Again, be sure to add SQL tests for `LIKE`. Make sure we support both wildcard chars (`_`, `%`). Be sure to also test for `LIKE` and `NOT LIKE`.

Reply:
I will fix LIKE in a separte PR